src/System.Memory/ref/System.Memory.cs

```diff
@@ -203,7 +203,7 @@ public static partial class SequenceReaderExtensions
         public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out int value) { throw null; }
         public static bool TryReadLittleEndian(this ref System.Buffers.SequenceReader<byte> reader, out long value) { throw null; }
     }
-    public ref partial struct SequenceReader<T> where T : struct, System.IEquatable<T>
+    public ref partial struct SequenceReader<T> where T : unmanaged, System.IEquatable<T>
```

Comment:
The tool doesn't support the `unmanaged` constraint yet.

Reply:
Is there a tracking bug for this and the other issues the tool has?

Reply:
For this one, yes: https://github.com/dotnet/arcade/issues/1299